### ISSUE TITLE
In the stallion tutorial, is it possible to predict on data for unknown agencies or skus?

### ISSUE THREAD

**Issue by tszumowski** [jdb78/pytorch-forecasting#511]
In the [Predict on new data](https://pytorch-forecasting.readthedocs.io/en/latest/tutorials/stallion.html#Predict-on-new-data) section of the [stallion tutorial](https://pytorch-forecasting.readthedocs.io/en/latest/tutorials/stallion.html), it generates future data to predict. This data is tied to each of the existing `{agency, sku}` groups. 

Suppose we have a new agency and a new sku, but we have 6 months of data on it. Is it possible to predict future months volume on this data? This is a case where there may not necessarily be enough data to re-train with this new agency/sku group. However, the hope would be that a model can learn trends from other groups that can then be applied to this data. 

In other words, can it predict on a short timeseries with unknown group IDs? I saw this as a (less strict) variant of "Cold-Start" listed in the [Model Architecture](https://pytorch-forecasting.readthedocs.io/en/latest/models.html?highlight=Cold-start#selecting-an-architecture) section, which has an `X` for the `TemporalFusionTransformer`, the model used in the stallion tutorial.

If it cannot predict for unknown groups, is there a model that is more appropriate for this? -- Thank you.

**Comment by jdb78** [comment]
Yes, you can. In case you do not use the `agency` or `sku` as a variable, just proceed. The TimeSeriesDataSet should take care of this. You can also use the `categorical_encoders={"sku": NaNLabelEncoder(add_nan=True)}` which will map unknown `sku`s effectively to a zero embedding.

**Comment by nagas1226** [comment]
I don't think that `categorical_encoders={"sku": NaNLabelEncoder(add_nan=True)}` works if there are some new skus.
Let's say there are 2 skus "A" and "B" which do not appear in the training set. In this case, both "A" and "B" are encoded to 0, resulting in regarding "A" and "B" as the same sku. 
How can I deal with this problem?
